src/components/home/Events.jsx

```diff
@@ -25,48 +24,40 @@ const Events = () => {
           </Typography>
           <Paper {...leftSidePaperStyles}>
             <Typography p={2}>
-              CODE PDX meets up in-person once a month for a project demo night and onboarding
-              event. We also occasionally do social meetups.
-              <br />
-              <br />
-              Not sure if we need three boxes. I Need to find more content to put here or remove a
-              bubble and maybe toss in a blob since this will be mid page.
+              CODE PDX meets up in-person twice a month at GRAYBOX in Portland. Once for a project
+              demo night and onboarding event and again for in person work session. We also
+              occasionally do non working social meetups. Event times and details can be found on
+              the calendar.
             </Typography>
           </Paper>
           <Paper
             elevation={3}
-            sx={{ ml: '20%', mb: '40px', borderRadius: '30px', background: '#d9d9d9' }}
+            sx={{
+              ml: '20%',
+              mb: '40px',
+              mr: '10%',
+              borderRadius: '20px',
+              background: 'rgba(217, 217, 217, 0.4)'
+            }}
           >
             <Typography p={2}>
               Individual projects often meet online weekly with schedules being determined per
               group. Additionally there is a biweekly in person work session once a month.
```

Comment:
I know the Meetup and Discord links are already in the footer but it might be nice to link to the Discord here as well.

Reply:
added